From d59f045a9341f33df161a83a0a5428e137381206 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Wed, 8 Dec 2021 05:47:11 +0900
Subject: [PATCH] dhcp: fix assertion failure

Fixes #21671.

(cherry picked from commit 990d0aa98023140d1efc897c3dcd5e0599a60203)

Conflict:NA
Reference:https://github.com/systemd/systemd/commit/d59f045a9341f33df161a83a0a5428e137381206
---
 src/libsystemd-network/sd-dhcp-lease.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libsystemd-network/sd-dhcp-lease.c b/src/libsystemd-network/sd-dhcp-lease.c
index 095a4ee683..9a0d7f6fea 100644
--- a/src/libsystemd-network/sd-dhcp-lease.c
+++ b/src/libsystemd-network/sd-dhcp-lease.c
@@ -691,7 +691,7 @@ int dhcp_lease_parse_options(uint8_t code, uint8_t len, const void *option, void
                 }
 
                 if (!timezone_is_valid(tz, LOG_DEBUG)) {
-                        log_debug_errno(r, "Timezone is not valid, ignoring: %m");
+                        log_debug("Timezone is not valid, ignoring.");
                         return 0;
                 }
 
-- 
2.33.0

